en/7-iterate-improve-frequently.md

```diff
@@ -32,390 +15,170 @@ altLangPage: 7-effectuer-iterations-ameliorations-constamment
-- Identify opportunities to improve the service based on the results of regular test **(2. Product management, not just project management. (Assess - Digital Design Playbook (ISED)))**
+- Testing
+  - Don’t wait for a fully developed service to start testing.  Develop a prototype of the service and test it to validate ideas, to challenge assumptions and to identify opportunities for improvement.
+  - Testing objective
+    - Define the purpose of the test and what you want to learn? The purpose of the test is often determined by your business goals and user needs identified through feedback, analytics and other sources.
+  - Testing audience
+    - You need to ensure that the service works from technical perspective and from the perspective of the user and the service provider (including the help desk agent who assists clients when they face challenges using the service). By testing with a diverse group and different type of users, you can capture a more comprehensive understanding of how your service is working.
+    - Make sure the participants are representative of your clients.
+    - Utilize user experience testing services offered by the Chief Information Office and the Communications team
+    - Create realistic scenarios that reflect the context and environment in which clients would use the service.
+  - Testing Methodology
+    - Identify the best testing method based on your needs. Examples of tests include:
+      - Tree Testing - A test in which participants are asked to find a resource based on a series of menus.
+      - Card Sorting Testing - A reverse tree test where participants sort through items and group them together in a hierarchal manner.
+      - First Click Testing - A test that observes the first item that a participant clicks on and uses the selection as an indication as to whether users are directed as intended.
+  - Analyse the Results
+    - identify and prioritize insights from user research
+    - analyze user research and use it to improve your service
+- Documentation is important
+  - document what you've built in each phase and why you built it (can be implemented in the style of a Change Log)
+  - document the lifecycle of a user story from user research to production
+- Service Deployment
+  - make sure you have the ability to deploy software frequently with minimal disruption to users
+  - determine how long you expect your service to be in beta and why
+  - document your way of [deploying software](https://www.gov.uk/service-manual/making-software/deployment.html), ie how you can deploy frequently with minimum impact on users
 
 </section>
 
-<section class="dpgn-section-guides">
-
-### Implementation guides
-
-**[TODO: Add/revise implementation guide items]**
-
-</section>
-
-<section class="dpgn-section-solutions">
+<section class="dpgn-section-similar">
 
-### Reusable solutions
+### Other resources
```

Comment:
This heading is managed centrally so changing it here won't have any effect (just provided here for convenience). It can be changed to something else if there is something better (would need to consult with the other areas first though).